Formula/gator.rb

```diff
@@ -0,0 +1,95 @@
+class Gator < Formula
+  desc "CLI Utility for Open Policy Agent Gatekeeper"
+  homepage "https://open-policy-agent.github.io/gatekeeper/website/docs/gator"
+  url "https://github.com/open-policy-agent/gatekeeper/archive/refs/tags/v3.9.0.tar.gz"
+  sha256 "af77ac7eedbe429e2b7df2f8470bc98d0af41a99f0829d95fc7883d34e23ba4d"
+  license "Apache-2.0"
+  head "https://github.com/open-policy-agent/gatekeeper.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/open-policy-agent/gatekeeper/pkg/version.Version=#{version}
```

Comment:
Let's check the output of `--version` in the `test` block to make sure that this `-X` flag is valid.

Reply:
currently that won't actually work - the tool itself is still outputting alpha as the version, but the intention is to utilize the ldflag in the future, so it's still passed in. I actually asked the maintainer about this as well and that's the answer for the moment